internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/VeniceParentHelixAdminTest.java

```diff
@@ -85,13 +86,13 @@ public void testTerminalStateTopicChecker() {
     properties.setProperty(TOPIC_CLEANUP_SLEEP_INTERVAL_BETWEEN_TOPIC_LIST_FETCH_MS, String.valueOf(Long.MAX_VALUE));
     properties.setProperty(TERMINAL_STATE_TOPIC_CHECK_DELAY_MS, String.valueOf(1000L));
     try (
-        VeniceControllerWrapper parentController = ServiceFactory.getVeniceParentController(
-            venice.getClusterName(),
-            zkServerWrapper.getAddress(),
-            venice.getKafka(),
-            new VeniceControllerWrapper[] { venice.getLeaderVeniceController() },
-            new VeniceProperties(properties),
-            false);
+        VeniceControllerWrapper parentController = ServiceFactory.getVeniceController(
+            new VeniceControllerCreateOptions.Builder(venice.getClusterName(), venice.getKafka())
+                .childControllers(new VeniceControllerWrapper[] { venice.getLeaderVeniceController() })
+                .zkAddress(zkServerWrapper.getAddress())
+                .extraProperties(properties)
+                .parent(true)
```

Comment:
Is `parent` necessary? When `childControllers` is present in the config, it must be a parent controller, right?

Reply:
Yes, it is redundant, but kept it mostly for readability. But we could certainly remove it. Let me know if you think we should remove it and soley rely on `childControllers` for the parent controller check. Either works for me.

Reply:
Let's remove it to avoid some confusion. :thumbsup:

Reply:
Sounds good. Removing it.